LinuxEX1.txt

```diff
@@ -0,0 +1,197 @@
+#firts exercise
+#a
+
+touch nameFile.txt
+cd nameFile.txt
+echo "antonna" > nameFile.txt
+cat nameFile.txt
+
+#b
+mkdir new
+
+#c
+cp ~/nameFile.txt ~/new
+cat new/nameFile.txt
+
+#d
+gedit ~/AntonNahhas.txt
+
+#e
+cd ~/new
+cp ../AntonNahhas.txt newFile.txt
+ls
+cat newFile.txt
+
+#f
+cd ~/new
+ls
+
+#g
+#first way
+cd /home
+#second way
+cd ~/new
+cd ..
+cd ..
+
+#h
+rm -f -r new
```

Comment:
you can combine the -f and -r options into a single -rf option to remove the new directory and its contents in one command.